### PR TITLE
media-libs/libjxl: version bump to 20220825 snapshot

### PR DESCRIPTION
**Pull request by novomesk**
This is latest commit from `v0.7.x` branch (so far). I think this is more reasonable choice than _0.7 release candidate_ which was released few weeks before (new commits solve problems that were discovered).

### CODE REVIEW

File: media-libs/libjxl/libjxl-0.7.0_pre20220825.ebuild [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	>=media-libs/lcms-2.13:=[${MULTILIB_USEDEP}]
+	media-libs/libpng:=[${MULTILIB_USEDEP}]
+	sys-libs/zlib[${MULTILIB_USEDEP}]
+	media-libs/libjpeg-turbo[${MULTILIB_USEDEP}]
+	>=x11-misc/shared-mime-info-2.2
+	openexr? ( media-libs/openexr:= )
+"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/libjxl-libjxl-4c31ef0"
```
Comment (ajakk) [inline media-libs/libjxl/libjxl-0.7.0_pre20220825.ebuild:31]: Move `S` to just after `SRC_URI`. While at it, maybe set `COMMIT=4c31ef03e3fd5239d6b794771d4ae7daa7815b28` so you can use it in `SRC_URI` then again in `S` as `${COMMIT:0:7}`?

File: media-libs/libjxl/libjxl-0.7.0_pre20220825.ebuild [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+DESCRIPTION="JPEG XL image format reference implementation"
+HOMEPAGE="https://github.com/libjxl/libjxl"
+
+SRC_URI="https://api.github.com/repos/libjxl/libjxl/tarball/4c31ef03e3fd5239d6b794771d4ae7daa7815b28 -> ${P}.tar.gz"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~riscv ~sparc ~x86"
+
+LICENSE="BSD"
+SLOT="0"
+IUSE="openexr"
```
Comment (ajakk) [inline media-libs/libjxl/libjxl-0.7.0_pre20220825.ebuild:16]: Make `LICENSE`, `SLOT`, `KEYWORDS`, `IUSE` their own block, in that order

**Review by ajakk — CHANGES_REQUESTED** [review]

File: media-libs/libjxl/libjxl-0.7.0_pre20220825.ebuild (hunk 2 above)
Comment (novomesk) [reply media-libs/libjxl/libjxl-0.7.0_pre20220825.ebuild:18]: Thanks for the review. Please take a look again if I understood correctly.

File: media-libs/libjxl/libjxl-0.7.0_pre20220825.ebuild [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+DEPEND="app-arch/brotli:=[${MULTILIB_USEDEP}]
+	dev-cpp/gflags:=[${MULTILIB_USEDEP}]
+	>=dev-cpp/highway-1.0.0[${MULTILIB_USEDEP}]
+	media-libs/giflib:=[${MULTILIB_USEDEP}]
+	>=media-libs/lcms-2.13:=[${MULTILIB_USEDEP}]
+	media-libs/libpng:=[${MULTILIB_USEDEP}]
+	sys-libs/zlib[${MULTILIB_USEDEP}]
+	media-libs/libjpeg-turbo[${MULTILIB_USEDEP}]
```
Comment (a17r) [inline media-libs/libjxl/libjxl-0.7.0_pre20220825.ebuild:27]: these are not alphanumerically ordered, otherwise lgtm

**Comment by a17r** [comment]
Hm, build failed here:
```
/usr/include/hwy/targets.h: In function ‘std::vector<unsigned int> hwy::SupportedAndGeneratedTargets()’:
<command-line>: error: ‘HWY_SVE_256’ was not declared in this scope; did you mean ‘HWY_SVE2’?
<command-line>: error: ‘HWY_SVE2_128’ was not declared in this scope; did you mean ‘HWY_SVE2’?
In file included from /usr/include/hwy/highway.h:24,
                 from /var/tmp/portage/media-libs/libjxl-0.7.0_pre20220825/work/libjxl-libjxl-4c31ef0/lib/jxl/dec_modular.cc:19,
                 from /usr/include/hwy/foreach_target.h:135,
                 from /var/tmp/portage/media-libs/libjxl-0.7.0_pre20220825/work/libjxl-libjxl-4c31ef0/lib/jxl/dec_modular.cc:18:
/usr/include/hwy/targets.h: In member function ‘size_t hwy::ChosenTarget::GetIndex() const’:
<command-line>: error: ‘HWY_SVE_256’ was not declared in this scope; did you mean ‘HWY_SVE2’?
<command-line>: error: ‘HWY_SVE2_128’ was not declared in this scope; did you mean ‘HWY_SVE2’?
```

**Review by a17r — APPROVED** [review]

**Comment by ajakk** [comment]
Sorry I missed this, you did indeed interpret me correctly, thanks for your work!

**Pull request closed without merging**